package-system/AWSNativeSDK/build_AWSNativeSDK_linux.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
Maybe copy this license (minus the bin bash line) to the other file?

Reply:
this also makes me think we probably SHOULD update the package becuase we do want the proper license :(

Reply:
package update will be done with version update (to 1.9) together after 3p migration.

Reply:
done